apps/backoffice-v2/src/pages/Entity/hooks/useTasks/useTasks.tsx

```diff
@@ -381,7 +382,11 @@ export const useTasks = ({
                     ...Object.entries(
                       omitPropsFromObject(entity?.data, 'additionalInfo', 'address') ?? {},
                     ),
-                    ...Object.entries(entity?.data?.additionalInfo ?? {}),
+                    ...Object.entries(
+                      workflow?.childWorkflows?.length
+                        ? omitPropsFromObject(entityDataAdditionalInfo ?? {}, 'ubos')
```

Comment:
Using `entityDataAdditionalInfo` instead of `entity.data.additionalInfo` since it does not include `mainContact` and so on.